File: DSA/LeetCode/Strings/1859. Sorting the Sentence/Solution.py
class Solution:
    def sortSentence(self, s: str) -> str:
        word_arr = s.split(' ')
        result_arr = [""] * len(word_arr)

        for i in range(len(word_arr)):
            word = word_arr[i]
            index = int(word[len(word)-1])
            result_arr[index-1] = word[:-1]

        return " ".join(result_arr)

File: DSA/LeetCode/Strings/1859. Sorting the Sentence/Solution.js
/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function (s) {
  const word_arr = s.split(' ')
  const result = new Array(word_arr.length).fill("")

  word_arr.map((word) => {
    const index = parseInt(word[word.length - 1])
    result[index - 1] = word.slice(0, -1);
  })
  return result.join(" ")
};